index.js

```diff
@@ -0,0 +1,80 @@
+// function takes in an integer and outputs a random integer between 0 & input
```

Comment:
I only see one file on your PR with a bunch of functions.... were you planning to implement these into a react application? it would be cool to see how you implement this file. :)

Reply:
Yeah that was the intention!  I'm not the fastest coder yet so I wanted to just the get the functionality done - then later implement it if I can!  Thanks!